liby2/src/genericfrontend.cc

```diff
@@ -62,7 +62,7 @@
  *
  * \section community Join the community
  *
- * Visit our web site at : http://en.opensuse.org/YaST or #yast on irc.freenode.net
+ * Visit our web site at : http://en.opensuse.org/YaST or #yast on irc.libera.chat
```

Comment:
Yes, but maybe even this?

```suggestion
 * Visit our web site at : http://en.opensuse.org/YaST or #yast on irc.opensuse.org
```
no such thing as a trivial PR!! :rofl:

Reply:
Well, it depends whether you use SSL or not. Without SSL `irc.opensuse.org` works fine. But with SSL the IRC client will complain about the certificate. There is an SSL certificate mismatch, the certificate was issued for `irc.libera.chat` but you are connecting to `irc.opensuse.org`...  :angry: 

See also https://github.com/yast/yast.github.io/pull/291#issuecomment-861715162